src/epcpm/anomaliestoxlsx.py

```diff
@@ -0,0 +1,277 @@
+import attr
+import openpyxl
+import typing
+import pathlib
+
+import epyqlib.pm.parametermodel
+import epyqlib.utils.general
+
+import epcpm.c
+import epcpm.pm_helper
+import epcpm.anomalymodel
+
+builders = epyqlib.utils.general.TypeMap()
+
+
+@attr.s
+class Fields(epcpm.pm_helper.FieldsInterface):
+    """The fields defined for a given row in the output XLS file."""
+
+    name = attr.ib(default=None, type=typing.Union[str, bool])
+    group = attr.ib(default=None, type=typing.Union[str, bool])
+    code = attr.ib(default=None, type=typing.Union[int, bool])
+    trig_type = attr.ib(default=None, type=typing.Union[int, bool])
+    resp_level_I = attr.ib(default=None, type=typing.Union[int, bool])
+    resp_level_A = attr.ib(default=None, type=typing.Union[int, bool])
+    comment = attr.ib(default=None, type=typing.Union[str, bool])
+
+
+field_names = Fields(
+    name="Name",
+    group="Group",
```

Comment:
In the spreadsheet, does it make more sense to have the Group be the first column and Name be second?

Reply:
Done